src/config/wallets.ts

```diff
@@ -138,6 +141,16 @@ export const supportEvmWalletObj = {
     isSupportMobileApp: true,
     ethExtension: 'ethereum',
   },
+  [SupportWallet.OneKey]: {
+    img: require('/src/assets/img/onekey.png'),
+    name: 'OneKey',
+    source: SupportWallet.OneKey,
+    walletUrl: 'https://onekey.so/download',
+    guideUrl: 'https://help.onekey.so/',
+    isSupportBrowserExtension: true,
+    isSupportMobileApp: true,
+    ethExtension: 'ethereum',
```

Comment:
`ethereum` is allocated to `MetaMask`. Selecting OneKey connects to MetaMask instead.

![image](https://user-images.githubusercontent.com/92044428/200739999-5de332c7-e1e4-4af1-b6a0-83d0a3471947.png)

Reply:
I think it is `$onekey.ethereum` instead of `ethereum`
![image](https://user-images.githubusercontent.com/92044428/200740483-181c1d1c-c1e3-459b-801b-e4f6a7e309e9.png)

Reply:
@loatheb 
btw, the [docs](
https://developer.onekey.so/guide/getting-started) have mentioned that calling `window.onekey` should be connected to API, but it returns undefined on my end. Do you aware of this issue?

![image](https://user-images.githubusercontent.com/92044428/200743413-3b357c99-5bf5-4b78-9c34-62f536c4f568.png)

![image](https://user-images.githubusercontent.com/92044428/200743511-0ff7b67d-be87-4d28-8976-6ba179f2e966.png)

Reply:
`window.onekey` is our legacy chrome extension. For our new monorepo APP, it's `window.$onekey.ethereum`, I have added `lodash.get` for getting new provider.